lib/Sema/CodeSynthesisDistributedActor.cpp

```diff
@@ -523,10 +525,10 @@ static FuncDecl *createDistributedThunkFunction(FuncDecl *func) {
       paramName = C.getIdentifier("p" + llvm::utostr(i));
     }
 
-    auto paramDecl = new (C)
-        ParamDecl(SourceLoc(),
-                  /*argumentNameLoc=*/SourceLoc(), funcParam->getArgumentName(),
-                  /*parameterNameLoc=*/SourceLoc(), paramName, DC);
+    auto paramDecl = new (C) ParamDecl(SourceLoc(),
+                               SourceLoc(), funcParam->getArgumentName(),
+                               SourceLoc(), funcParam->getParameterName(),
```

Comment:
This has to use `paramName`, the parameter cannot be anonymous if it's used in the body.

Reply:
Oh, thanks for catching that

Reply:
I messed up the rebase picking the wrong impl here it seems, thanks